paymentsheet/src/main/java/com/stripe/android/common/model/CommonConfiguration.kt

```diff
@@ -25,6 +29,7 @@ internal data class CommonConfiguration(
     val externalPaymentMethods: List<String>,
     val cardBrandAcceptance: PaymentSheet.CardBrandAcceptance,
 ) : Parcelable {
+    @Suppress("LongMethod", "ThrowsCount")
```

Comment:
rather than suppressing these can you extract some of the functionality of this method into a helper function or two?